tests/post_training/data/ptq_reference_data.yaml

```diff
@@ -34,6 +34,14 @@ torchvision/resnet18_backend_CUDA_TORCH:
   metric_value: 0.69152
 torchvision/resnet18_backend_FX_TORCH:
   metric_value: 0.6946
+torchvision/mobilenet_v3_small_backend_FP32:
+  metric_value: 0.6766
+torchvision/mobilenet_v3_small_backend_OV:
+  metric_value: 0.6669
+torchvision/mobilenet_v3_small_backend_ONNX:
+  metric_value: 0.6654
+torchvision/mobilenet_v3_small_backend_FX_TORCH:
```

Comment:
Model | Backend | Num FQ | Num int8
-- | -- | -- | --
torchvision/mobilenet_v3_small | FX_TORCH | 61 | 36
torchvision/mobilenet_v3_small | OV | 79 | 54


What is the reason that number of int8 weights and FQ is less that for OV?